pymc3/tests/test_advi.py

```diff
@@ -2,7 +2,8 @@
 import pymc3 as pm
 from pymc3 import Model, Normal, DiscreteUniform, Poisson, switch, Exponential
 from pymc3.theanof import inputvars
-from pymc3.variational.advi import variational_gradient_estimate, advi, advi_minibatch, sample_vp
+from pymc3.variational import advi, advi_minibatch, sample_vp
```

Comment:
I checked that the below notebooks work with small fixes. 

* docs/source/notebooks/bayesian_neural_network_advi.ipynb
* docs/source/notebooks/gaussian-mixture-model-advi.ipynb
* docs/source/notebooks/GLM-hierarchical-advi-minibatch.ipynb